python/requirements.in/development.txt

```diff
@@ -7,4 +7,6 @@ pep8-naming
 mypy
 pip-tools
 pytest
-types-Deprecated
\ No newline at end of file
+types-Deprecated
+openapi-python-client
+types-python-dateutil
```

Comment:
as tom rightly pointed out (this was my mistake), the type libraries need to be actual dependencies (not dev dependencies), i believe this also means they need to be added to the setup.py so they get autoinstalled for our users (but @tasn correct me if im wrong)

Reply:
Just the setup.py file needs updating in that case? Or also requirements.txt?